src/Command/ListMissingDownloadsCommand.php

```diff
@@ -33,24 +33,28 @@
 
 class ListMissingDownloadsCommand extends Command
 {
+    /**
+     * @var int
+     */
     private const FORMAT_TAR = 0;
+
+    /**
+     * @var int
+     */
     private const FORMAT_ZIP = 1;
 
+    /**
+     * @var string
+     */
     protected static $defaultName = 'app:download:missing:list';
 
     protected function configure()
     {
-        $this
-            // the short description shown while running "php bin/console list"
-            ->setDescription('Creates a list of missing downloads with URLs to Sourceforge.')
-
-            // the full command description shown when running the command with
-            // the "--help" option
-            ->setHelp('This command allows you to create a list of missing downloads with URLs to Sourceforge if available there.')
-        ;
+        $this->setDescription('Creates a list of missing downloads with URLs to Sourceforge.');
+        $this->setHelp('This command allows you to create a list of missing downloads with URLs to Sourceforge if available there.');
```

Comment:
I'm not sure I agree with Rector that this is better.  It's not wrong, I just disagree about whether it's "better".

Reply:
Yeah, was also wondering about this change. But well, the comments aren't really helpful so I tend to agree.